backends/ebpf/tests/p4testdata/meters-direct-and-counter.p4

```diff
@@ -0,0 +1,191 @@
+/*
+Copyright 2022-present Orange
+Copyright 2022-present Open Networking Foundation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <core.p4>
+#include <psa.p4>
+
+typedef bit<48>  EthernetAddress;
```

Comment:
I thought you were using some common definitions of the headers using an #include.
Forgot to check this in a few previous PRs, perhaps you want to scan the recent test additions.